watches.yaml

```diff
@@ -0,0 +1,9 @@
+---
+- version: v1alpha1
```

Comment:
we are not using alpha version.

Reply:
Alpha is referred to Astarte Kubernetes' CRD revision, so it's correct, and it's Kubernetes standard way of doing things

Reply:
So v0.10.0-beta.3 is v1alpha1? So how is going to be called v0.10.0?